hydra-node/test/Hydra/Chain/Direct/Contract/Close.hs

```diff
@@ -148,6 +148,7 @@ healthySnapshot =
     , number = healthyCloseSnapshotNumber
     , utxo = healthyCloseUTxO
     , confirmed = []
+    , utxoToDecommit = Nothing
```

Comment:
why not arbitrary?

Reply:
Adding this field just to remove the compiler warning. We will test thoroughly the decommit workflow with respect to `Snapshot` type.